srcpkgs/lsd/template

```diff
@@ -12,7 +12,7 @@ license="Apache-2.0"
 homepage="https://github.com/lsd-rs/lsd"
 changelog="https://raw.githubusercontent.com/Peltoche/lsd/master/CHANGELOG.md"
```

Comment:
This file no longer seems to exist, do you think it was deleted accidentally upstream?

Reply:
Thanks for catching that! It was removed because the changelog was unmaintained: https://github.com/lsd-rs/lsd/commit/16181df25b3df555833b2f5f7e804b49b86515c1#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edL3

I'll point the template to the release page